README.md

```diff
@@ -90,7 +90,7 @@ additional resources.
 * `openssl`
 
 ## Bootstrapping the Cluster
-
+Run git clone of the konflux-ci repo ```git clone https://github.com/konflux-ci/konflux-ci.git``` 
```

Comment:
To fit with the existing formatting, I think it should look something like what I put below. Also, I think this step should be enumerated, which means we should increment the numbers of steps 1-5 after it.

```suggestion
:gear: Clone this repository:

 ```bash
git clone https://github.com/konflux-ci/konflux-ci.git
```
```